docs/upgrade/focal_migration.rst

```diff
@@ -0,0 +1,240 @@
+Ubuntu 20.04 LTS (Focal) Migration Guide
+========================================
+
+On April 30, 2021, Ubuntu 16.04 LTS (Xenial), the operating system for the
+SecureDrop servers, will reach End of Life. In order to continue using SecureDrop, 
+instances must migrate to Ubuntu 20.04 LTS (Focal) **before** April 30, 2021.
+
+This migration will require on-premises access to the servers, and a complete
+reinstallation of Ubuntu and SecureDrop. In-place upgrades and 
+remote upgrades via SSH are not currently supported.
+
+.. important::
+
+   For security reasons, the *Source Interface* will automatically be
+   disabled on SecureDrop servers still running Ubuntu 16.04 after 
+   April 30, 2021.
+
+We recommend that you plan a two day maintenance window
+**between March 9 and April 30** to perform and test the migration.
+
+At a high level, the migration process consists of:
+
+- Taking a backup of the current instance
+- Installing Ubuntu 20.04 LTS and SecureDrop from scratch
+- Applying the backup
+
+Instances that already have :doc:`v3 onion services <../v3_services>` enabled 
+and follow our migration guide will be able to 
+preserve their existing *Source* and *Journalist Interface* onion URLs.
```

Comment:
As a reader, that left me with the question "what if I'm not running v3?". I know this is clarified later in the doc, but maybe add another sentence here, e.g.:
> If you are still running v2 onion services, you must switch to v3 as part of the migration at the latest, and your onion addresses will change.